### PR TITLE
MINOR: update producer client request timeout in system test

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
ping @guozhangwang updated producer client request timeouts to 60S

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8050 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8050 tests run, 5 skipped, 0 failed.
--none--

File: streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java [hunk 1]
```diff
@@ -232,6 +232,7 @@ public Properties setStreamProperties(final String applicationId) {
         props.put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, MAX_POLL_RECORDS);
         props.put(StreamsConfig.POLL_MS_CONFIG, POLL_MS);
         props.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, COMMIT_INTERVAL_MS);
+        props.put(StreamsConfig.producerPrefix(ProducerConfig.REQUEST_TIMEOUT_MS_CONFIG), 60000);
```
Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java]: I think we should leave a comment to remove this changes or set to smaller values when KIP-91 is merged in. Ditto elsewhere

Comment (bbejeck) [reply streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java]: ack

**Comment by bbejeck** [comment]
updated this

**Comment by guozhangwang** [comment]
@bbejeck seems one of my previous comment was dropped somehow: did you trigger a system test build to confirm no regression is introduced?

**Comment by asfgit** [comment]
FAILURE
 No test results found.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8050 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8050 tests run, 5 skipped, 0 failed.
--none--

**Comment by bbejeck** [comment]
Systems test passed [results link](http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2017-11-02--001.1509654736--bbejeck--MINOR_update_streams_produer_timeout_in_system_test--20b3c2c/report.html)

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and 1.0.

**Pull request closed without merging**